### ISSUE TITLE
Unable to propose a merge outside mainspace

### ISSUE THREAD

**Issue by NovemLinguae** [wikimedia-gadgets/twinkle#1932]
Reported by Sdkb at https://en.wikipedia.org/wiki/Wikipedia_talk:Twinkle#Merging_help%2Fproject_pages

> It looks like there's currently no way to nominate pages in the Help or Wikipedia namespaces for merging, because the merge function is under the tag menu, which appears only for articles. That's unfortunate, given that there are [plenty of projectspace pages](https://en.wikipedia.org/wiki/Wikipedia:Avoid_writing_redundant_essays) that could use merging, so we want to make it easy to suggest doing so. --Sdkb

One fix would be to move merge stuff from the Tag module into the XFD module, where things like AFD and RM are located.

**Comment by sdkb** [comment]
Thanks for creating the ticket!

I like the idea of moving to the XfD module, as merging seems more akin to the other stuff there (all of which is in the "affects the page structure of the encyclopedia" bucket) than it does to maintenance tags, even if it's not technically an XfD.

Before we go ahead with that, it is probably worth checking to see if anything else in the tag menu is applicable outside mainspace, since if so we'll have different considerations.

**Comment by Gonzom** [comment]
@sdkb , yes  #1916 .

**Comment by FaviFake** [comment]
This should be closed as not done. The PAM process has been would down and there is no longer a formal process for proposing a merge of two projectspace pages. The old {{merge}} tags can no longer be used outside articlespace.

**Comment by sdkb** [comment]
So, just to make sure I understand, what is the current suggested workflow for nominating a non-article page for merging via Twinkle? For articles, I can AfD it and choose the "desired outcome=merge" option, but there's no option to AfD something in e.g. userspace, and the MfD dialogue doesn't have a merge option. So we need to add it there? If so, I wouldn't consider this totally resolved.

**Comment by NovemLinguae** [comment]
When I closed this, I assumed there is no formal process for merging in Wikipedia space. Is there a process? If so, what do you suggest the workflow be? Like, what tags or processes would Twinkle use to indicate that a merge is desired? (I assume MFD is not used for merging, but correct me if I'm wrong.)

**Comment by sdkb** [comment]
From [WP:Merging](https://en.wikipedia.org/wiki/Wikipedia:Merging#Merging_pages_in_non-article_namespaces):

> Pages in certain other [namespaces](https://en.wikipedia.org/wiki/Wikipedia:Namespace) can also be merged, usually also on a bold basis, but merging on a consensus basis relies on different venues. For most non-article namespaces, the counterpart to AfD is [MfD](https://en.wikipedia.org/wiki/Wikipedia:MFD); unlike the former, it does not accept merge nominations, though a deletion discussion can yield "merge" as an [alternative to deletion](https://en.wikipedia.org/wiki/Wikipedia:ATD-M). Templates can be explicitly nominated for merging at [TfD](https://en.wikipedia.org/wiki/Wikipedia:TFD). The [CfD](https://en.wikipedia.org/wiki/Wikipedia:CFD) process governs category merges; these fall entirely outside the scope of this page because "merging" categories is not merging in the usual sense (and is not done boldly either).

**Comment by sdkb** [comment]
MfD not formally accepting merges seems like a hole, as that means there's no consensus-based process for nominating e.g. project-space pages for merging. I could raise that on-wiki if you think it'd be a good idea?

**Comment by NovemLinguae** [comment]
The quoted text seems to support keeping this ticket closed. Let me know if you're interpreting it differently. Basically, what would we code up in Twinkle? The plan is unclear.

Sure, feel free to explore the answer to this question onwiki.

**Comment by sdkb** [comment]
Agreed. I'll raise it on-wiki and follow up if there are changes to the process.

**Comment by FaviFake** [comment]
Both of you are correct. PAM was the only formal process for proposing a projectspace/draftspace merge, and now that it's gone there simply is no longer any formal process for those namespaces. This issue was briefly discussed [here](https://en.wikipedia.org/wiki/Wikipedia_talk:Articles_for_deletion/Implementing_the_PAM%E2%80%93AfD_merge#Where_are_miscellaneous_merges_discussed?), but I think a new broader discussion would be beneficial. I would strongly support renaming MfD to "Miscellany for discussion" and allowing merge nominations like any other XfD venue. Paradoxically, one can now formally nominate articles to be merged into a draft, but can't nominate a draft to be merged into an article or another draft.

**Comment by sdkb** [comment]
Started; see https://en.wikipedia.org/wiki/Wikipedia_talk:Miscellany_for_deletion#Accepting_merge_proposals_at_MfD. Cheers!
